src/main/scala/org/broadinstitute/hail/driver/Main.scala

```diff
@@ -35,8 +35,12 @@ object SparkManager {
 
       conf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
       _sc = new SparkContext(conf)
-      _sc.hadoopConfiguration.set("io.compression.codecs",
-        "org.apache.hadoop.io.compress.DefaultCodec,org.broadinstitute.hail.io.compress.BGzipCodec,org.apache.hadoop.io.compress.GzipCodec")
+      if(gzAsBgz)
+        _sc.hadoopConfiguration.set("io.compression.codecs",
+          "org.apache.hadoop.io.compress.DefaultCodec,org.broadinstitute.hail.io.compress.BGzipCodec, org.broadinstitute.hail.io.compress.BGzipCodecGZ")
```

Comment:
I think we really want it on ImportVCF, right?  I think you can make that happen by copying the spark context and setting the io.compression.codecs on the copy, then using that